tests/e2e/support/objects/app-files/share/actions.ts

```diff
@@ -187,10 +193,17 @@ export const hasPermissionToShare = async (
   return !(await page.isVisible(noPermissionToShareLabel))
 }
 
-export const copyQuickLink = async (args: copyLinkArgs): Promise<void> => {
+export const copyAndGetQuickLinkUrl = async (args: copyLinkArgs): Promise<string> => {
   const { page, resource, via } = args
+  let url = null
   if (via === 'CONTEXT_MENU') {
     await clickActionInContextMenu({ page, resource }, 'create-quicklink')
+    if (config.ocis) {
+      url = await page.evaluate(() => navigator.clipboard.readText())
+    } else {
+      url = await page.locator(util.format(publicLinkInputField, 'Quicklink')).textContent()
+    }
```

Comment:
Separating the logic here for oc10 and ocis because according to MDN [Clipboard](https://developer.mozilla.org/en-US/docs/Web/API/Clipboard) docs
> This feature is available only in [secure contexts](https://developer.mozilla.org/en-US/docs/Web/Security/Secure_Contexts) (HTTPS), in some or all [supporting browsers](https://developer.mozilla.org/en-US/docs/Web/API/Clipboard#browser_compatibility).

In case of `oc10` the values is directly read from the input field